src/components/Animatable.scss

```diff
@@ -0,0 +1,24 @@
+@import './Animation.scss';
+
+.Animatable {
+  animation-duration: var(--transition-duration-medium);
+  animation-fill-mode: both;
+}
+
+.ZoomInUp {
+  animation-name: zoomInUp;
+}
+
+@keyframes zoomInUp {
+  from {
+    opacity: 0;
+    transform: scale3d(0.5, 0.5, 0.5) translate3d(0, 1000px, 0);
+    animation-timing-function: cubic-bezier(0.55, 0.055, 0.675, 0.19);
```

Comment:
These are pretty specific at the moment, and so I've elected to not share them yet in Animation.scss. If I start to see a repeatable pattern I will, but for now this is one-off